centreon/src/Core/HostCategory/Infrastructure/Repository/DbReadHostCategoryRepository.php

```diff
@@ -259,10 +259,54 @@ public function existsByAccessGroups(int $hostCategoryId, array $accessGroups):
         foreach ($concat->retrieveBindValues() as $param => [$value, $type]) {
             $statement->bindValue($param, $value, $type);
         }
-        $statement->bindValue(':hostCategoryId', $hostCategoryId, \PDO::PARAM_INT);
+        $statement->bindValue(':hostCategoryId', $hostCategoryId, \PDO::PARAM_STR);
```

Comment:
The variable is an `integer`, same in database I guess, why using `PARAM_STR` ?

Reply:
No idea 😅 , fixed